paper-drawer-panel.html

```diff
@@ -289,13 +291,17 @@
         activate-event=""
         selected="[[selected]]">
 
-      <div id="main" style$="[[_computeMainStyle(narrow, rightDrawer, drawerWidth)]]" on-transitionend="_onMainTransitionEnd">
+      <div id="main" style$="[[_computeMainStyle(narrow, rightDrawer, drawerWidth)]]">
         <content select="[main]"></content>
         <div id="scrim" on-tap="closeDrawer"></div>
       </div>
 
       <div id="drawer" style$="[[_computeDrawerStyle(drawerWidth)]]">
+        <!-- first tab stop -->
+        <div tabindex="0" id="firstTabStop" hidden$="[[_isDrawerClosed(narrow, selected)]]"></div>
         <content select="[drawer]"></content>
+        <!-- last tab stop -->
+        <div tabindex="0" id="lastTabStop" hidden$="[[_isDrawerClosed(narrow, selected)]]"></div>
```

Comment:
Should we prevent the user from focusing the address bar? Even when the drawer is open, it doesn't "cover" the address bar, and IMO the user should still be able to focus it (e.g. to navigate to a different site).